src/helpers/calculateRequiredKcurTokens.ts

```diff
@@ -0,0 +1,50 @@
+// @ts-ignore
+import * as ethers from "ethers";
+import { BigNumber, BytesLike, Contract } from "ethers";
+
+async function calculateRequiredkCur(
```

Comment:
this should be moved into the floor-ceiling service unless it is reusable, which it isn't

Reply:
Done